src/mock-graphql/typeDefs.ts

```diff
@@ -7,6 +7,12 @@ export const typeDefs = gql`
         updated: String!
     }
 
+    type User {
```

Comment:
Do we want this to be a `Profile` type in anticipation of the mock `People` we'll need for the people picker query?